init.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Do you want this script to be executable so that users who clone our repo don't need to `chmod +x`?

Reply:
Apoorva pulled this branch and ran the script without changing permissions. I'm not very familiar with scripting, so if the addition makes the process more streamlined I'm good with it.

Reply:
I guess he ran something like `bash init.sh` which doesn't require executable permission, but `./init.sh` does need it.

Reply:
I think leaving it as-is for now is fine, it can be changed very easily in the future if needed.